context/.eslintrc.yml

```diff
@@ -10,6 +10,7 @@ settings:
   react:
     version: detect
 rules:
+  no-console: ["error", { allow: ["warn", "error"] }]
```

Comment:
I would be in favor of making less of an effort to handle all the variants we see (since it's unclear now which are intended, and which are accidents)... but when there is something weird log an error close to where it happens.

Reply:
I agree it makes sense for now.